lib/vpc/manager/Manager.rb

```diff
@@ -88,7 +91,112 @@ def update(local, diffs)
             end
           end
         end
+      end
+
+      def migrate
+        # Create the directories
+        vpc_dir = "#{@migration_root}/vpc"
+        policies_dir = "#{vpc_dir}/policies"
+        route_tables_dir = "#{vpc_dir}/route-tables"
+        subnets_dir = "#{vpc_dir}/subnets"
+        vpcs_dir = "#{vpc_dir}/vpcs"
+
+        if !Dir.exists?(@migration_root)
+          Dir.mkdir(@migration_root)
+        end
+        if !Dir.exists?(vpc_dir)
+          Dir.mkdir(vpc_dir)
+        end
+        if !Dir.exists?(policies_dir)
+          Dir.mkdir(policies_dir)
+        end
+        if !Dir.exists?(route_tables_dir)
+          Dir.mkdir(route_tables_dir)
+        end
+        if !Dir.exists?(subnets_dir)
+          Dir.mkdir(subnets_dir)
+        end
+        if !Dir.exists?(vpcs_dir)
+          Dir.mkdir(vpcs_dir)
+        end
+
+        # Migrate the different assets
+        migrate_policies(policies_dir)
+        route_table_names = migrate_route_tables(route_tables_dir)
+        subnet_names = migrate_subnets(subnets_dir, route_table_names)
+        migrate_vpcs(vpcs_dir, route_table_names, subnet_names)
+      end
+
+      # Public: Migrates the endpoint policies. Uses Version to name the file
+      #
+      # dir - the directory to store policies in
+      def migrate_policies(dir)
+        puts "Migrating policies to #{dir}"
+        # Get the policies for each endpoint
+        policies = EC2::endpoints.map { |endpoint| endpoint.parsed_policy }
+        policies.each do |policy|
+          name = policy["Version"]
```

Comment:
Don't have much else to go off of. It's only possible to have one service endpoint for now (the whole thing was kind of half-baked it seems) so I don't anticipate there being more than just the default policy with the same version